lambdatimes/src/components/Content/Content.js

```diff
@@ -37,7 +42,10 @@ export default class Content extends Component {
           of the items from cardData. 
         - else, it should only return those cards whose 'tab' matched this.state.selected.
     */
-    return this.state.cards;
+    let filteredCards = this.state.cards.slice().filter(card => {
+      return card.tab === this.state.selected;
```

Comment:
filter will return a new array so you dont need to do the slice here, but good job chaining the methods though